src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.NuGet.targets

```diff
@@ -10,5 +10,25 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project>
-  <Import Project="..\targets\Microsoft.NET.Sdk.targets" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.Shared.UsingNetSdkDefaults.props" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.Shared.UseCurrentRuntime.props" />
+  
+  <ImportGroup>
+    <Import Project="..\targets\Microsoft.PackageDependencyResolution.targets" Condition="Exists('..\targets\Microsoft.PackageDependencyResolution.targets') and ('$(Language)' != 'C++' or '$(_EnablePackageReferencesInVCProjects)' == 'true')" />
+  </ImportGroup>
+
+  <Import Project="..\targets\Microsoft.NET.Sdk.DefaultItems.props" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.DefaultItems.Shared.props" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.DefaultItems.Shared.targets" />
+
+  <Import Project="..\targets\Microsoft.NET.Sdk.FrameworkReferenceResolution.targets" />
+  
+  <Import Project="..\targets\Microsoft.NET.Sdk.Shared.props" />
+  <Import Project="..\targets\Microsoft.NET.Sdk.Shared.targets" />
```

Comment:
Same here, put the contents of both in `Microsoft.NET.Sdk.Shared.targets`.

Reply:
Done. Thanks.